Shared/DsaResources.cpp

```diff
@@ -45,6 +45,24 @@ DsaResources::~DsaResources()
 { 
 }
 
+/*!
+ * \brief Sets the string for the Maps SDK version from main.cpp
+ * \param const char* from the ARCGIS_MAPS_SDK_VERSION define
+ */
+void DsaResources::setArcGISMapsSDKVersion(const char *arcGISMapsSDKVersion)
+{
+  m_arcGISMapsSDKVersion = QString(arcGISMapsSDKVersion);
+}
+
+/*!
+ * \brief Returns the version string of the Maps SDK that is used by the project
+ * \return
```

Comment:
This looks incomplete, but I don't know if we actually run qdoc on this project anymore.
@ldanzinger do you know about that?

Reply:
I removed the return statement here. Perhaps it's good enough for now to just include a *brief* for methods?

Reply:
Probably fine for now.